go.mod

```diff
@@ -8,10 +8,10 @@ require (
 	github.com/coreos/go-systemd/v22 v22.1.0
 	github.com/frankban/quicktest v1.9.0 // indirect
 	github.com/gardener/etcd-druid v0.3.0
-	github.com/gardener/gardener v1.19.1-0.20210401122359-e16e6ec0002e
+	github.com/gardener/gardener v1.20.1
 	github.com/gardener/machine-controller-manager v0.36.0
 	github.com/go-logr/logr v0.3.0
-	github.com/gobuffalo/packr/v2 v2.8.1
+	github.com/gobuffalo/packr/v2 v2.5.1
```

Comment:
packr is still in go.mod because there is a still reference in `hack/tools.go`

Reply:
Thanks for catching!